csharp/ql/src/semmle/code/csharp/controlflow/internal/Completion.qll

```diff
@@ -668,165 +658,142 @@ abstract private class InheritableCompletion extends AbnormalCompletion { }
  * `System.Console.WriteLine("M called")` inherits the throw completion
```

Comment:
We should remove the word "inherits" and "underlying" and replace it with "has the outer" and "has the inner"